crates/bevy_ecs/src/component.rs

```diff
@@ -84,6 +84,21 @@ use std::{
 /// [`Table`]: crate::storage::Table
 /// [`SparseSet`]: crate::storage::SparseSet
 ///
+/// # Disabling Change Detection
+///
+/// By default, Bevy will track every mutative access made to a given component.
+/// This may incur a performance cost on types that do not need it.
```

Comment:
```suggestion
/// This may incur a performance cost (primarily in terms of memory usage) on types that do not need it.
```

Reply:
Not sure I agree with this when it's currently blocking autovectorization, as seen in #6547. It's both a CPU and a memory cost in that case.